File: Java/Logica-Programacao/src/faccat/Ex011.java
package faccat;

import java.util.Scanner;

public class Ex011 {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Digite o salário fixo do funcionário: R$");
        float salary = sc.nextFloat();
        System.out.print("Digite quantos carros foram vendidos pelo funcionário: ");
        short qtyCar = sc.nextShort();
        System.out.print("Digite o valor da comissão fixa por carro vendido: R$");
        float commission = sc.nextFloat();
        System.out.print("Digite o valor total das vendas do funcionário: R$");
        float sellsTotalPrice = sc.nextFloat();
        sc.close();

        float totalSalary = salary + (commission * qtyCar) + (5 * sellsTotalPrice / 100);

        System.out.printf("O funcionário vai receber ao todo R$%.2f", totalSalary);
    }
}

File: Logica-Programacao/Javascript/Faccat/ex011.js
let salario, carrosVendidos, valorVendas, valor;

salario = parseFloat(prompt("Digite o salário(R$) fixo do funcionário: "));
carrosVendidos = parseInt(prompt("Digite o número de carros vendidos por esse funcionário: "));
valor = parseFloat(prompt("Digite o valor(R$) recebido por venda realizada: "));
valorVendas = parseFloat(prompt("Digite o valor(R$) total das vendas realizadas por esse funicionário: "));

salario = salario + (carrosVendidos * valor) + (valorVendas * 0.05);

alert("Salário final: R$" + salario);